src/python/grpcio_tests/tests/qps/worker_server.py

```diff
@@ -91,13 +92,14 @@ def _create_server(self, config):
             raise Exception('Unsupported server type {}'.format(
                 config.server_type))
 
+        server_port = config.port if self._server_port is None else self._server_port
```

Comment:
This is actually a different behavior than in grpc-java.
In java, ServerConfig.port takes precedence

https://github.com/grpc/grpc-java/blob/59528d8efec465326b071ffc6f3f2220bcf641c9/benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadWorker.java#L136

Reply:
Let's fix this (and make the flag's documentation clear).

Reply:
@jtattermusch I do not think this behavior is consistent across languages. I based this change off of the C++ worker. In my understanding, it lacks the second condition which checks that the port is unspecified in the scenario. In essence, it overrides when the flag is set:

https://github.com/grpc/grpc/blob/04045e227359ae23b5b31fb9e7d14e3b79d3e373/test/cpp/qps/qps_worker.cc#L237-L239

Reply:
@codeblooded  good observation! so yeah, currently python and C++ are inconsistent with the behavior in go and java.
I think this is something that should be fixed since the qps_workers are supposed to be interchangeable (so that the infrastructure doesn't need to care which language's qps worker you're using). Currently the --port_server flag behavior is inconsistent, which breaks that invariant.

Reply:
Agreed. 👍 

Since we have control of generating the scenarios in the python script, would you want us to include a port in each scenario (making it unnecessary for our system to include this flag)? Or, would you rather us leave it as it is (and continue adding it to workers consistently)?

Reply:
I created a PR to fix C++'s behavior. @codeblooded can you  create a PR to update the python qps_worker behavior?